signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/OrganizerSignupMBean.java

```diff
@@ -443,6 +448,15 @@ private SignupEventTrackingInfo moveAttendee() throws Exception {
 
 	}
 
+	private static User getUser(String userId) {
+		try {
+			return UserDirectoryService.getUser(userId);
```

Comment:
Why are you using a cover here? This bean already uses dependency injection in faces-config. Is it for performance reasons?